radareorg/cutter
# Rename "Open " to "Map File" and "New" to "New Window"

**Your checklist for this pull request**


**Detailed description**

Multiple times in the past I noticed people that are getting confused by the "Open" menu item in File->Open. It is mistaken to be an item to open a new file in Cutter, while the truth is that it is used to map a file to the IO of the same session.

Thus, I renamed it to `Map File` and renamed `New` to `New Window`.
In addition, I changed the source names from OpenFileDialog to MapFileDialog to reflect the reality.

**Test plan (required)**

1. Open Cutter, no need to open a file
2. Choose File->New Window and see that a new window of cutter is opened
3. Use <kbd>Ctrl</kbd>+<kbd>N</kbd> and see the same
4. Choose File->Map File and see the map new file dialog
5. Use <kbd>Ctrl</kbd>+<kbd>M</kbd> and see the same
